arviz/plots/plot_utils.py

```diff
@@ -695,3 +695,25 @@ def _dealiase_sel_kwargs(kwargs, prop_dict, idx):
             {prop: props[idx] for prop, props in prop_dict.items()}, "plot"
         ),
     }
+
+
+def is_valid_quantile(value):
+    """Check if value is a number between 0 and 1."""
+    try:
+        value = float(value)
+        return 0 < value < 1
```

Comment:
I think you want this return _after_ the except, since you are watching for a ValueError while casting to float, right?

Reply:
I think this works ok.

Reply:
This will work as written